NEO-TAT/parrot_number
# Implement additional features

## Description

This pull request implements the following features
- Show a snack bar when user inputs invalid string
- Implement share button
- Change Launcher Icon
- Add Hero Animation to `ParrotGif`
- Change font
- Add `Text` and `Button` animations

## How to verify

1. Verify if the features listed above works fine
2. Check if there are no weird actions in project
